metricbeat/module/prometheus/remote_write/_meta/docs.asciidoc

```diff
@@ -7,6 +7,23 @@ remote_write:
   - url: "http://localhost:9201/write"
 ------------------------------------------------------------------------------
 
+
+TIP: In order to assure the health of the whole queue, the following two configuration
+ https://prometheus.io/docs/practices/remote_write/#parameters[parameters] should be considered:
+
+- `max_shards`: Sets the maximum number of paralelism with which Prometheus will try to send samples to Metricbeat.
```

Comment:
I know this is a cherry-pick, I did not review the original PR, but `paralelism` should be `parallelism`.

Reply:
👍